node/internal/util/types.ts

```diff
@@ -22,7 +22,6 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 import * as bindingTypes from "../../internal_binding/types.ts";
-export { isCryptoKey, isKeyObject } from "../crypto/keys.ts";
```

Comment:
Causes circular references. We can reference the crypto module directly if needed.